src/utils/API.js

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+
+const API = axios.create({
+  baseURL: "https://gotogro-msp20-api-test.herokuapp.com",
+  // baseURL: "http://localhost:4000",
+  responseType: "json",
+});
+
+export const addMember = async (memberToAdd, successModalRef, errorModalRef) => {
+  try {
+    const response = await API.post("/addmember", {
```

Comment:
We'll do a `post` request, passing in the member details to `https://gotogro-msp20-api-test.herokuapp.com/addmember`